include/z64item.h

```diff
@@ -295,27 +299,58 @@ typedef enum {
     /* 0x22 */ GI_QUIVER_30,
     /* 0x23 */ GI_QUIVER_40,
     /* 0x24 */ GI_QUIVER_50,
-    /* 0x25 */ GI_25 = 0x25,
-    /* 0x28 */ GI_NUTS_1 = 0x28,
-    /* 0x2A */ GI_NUTS_10 = 0x2A,
+    /* 0x25 */ GI_ARROW_FIRE,
+    /* 0x26 */ GI_ARROW_ICE,
+    /* 0x27 */ GI_ARROW_LIGHT,
+    /* 0x28 */ GI_NUTS_1,
+    /* 0x29 */ GI_NUTS_5,
+    /* 0x2A */ GI_NUTS_10,
+    /* 0x2E */ GI_BOMBCHUS_20 = 0x2E,
     /* 0x32 */ GI_SHIELD_HERO = 0x32,
     /* 0x33 */ GI_SHIELD_MIRROR,
-    /* 0x35 */ GI_MAGIC_BEANS = 0x35,
+    /* 0x34 */ GI_POWDER_KEG,
+    /* 0x35 */ GI_MAGIC_BEANS,
     /* 0x36 */ GI_BOMBCHUS_1,
-    /* 0x3A */ GI_BOMBCHUS_5 = 0x3A,
-    /* 0x3C */ GI_KEY_SMALL = 0x3C,
+    /* 0x37 */ GI_SWORD_KOKIRI,
+    /* 0x38 */ GI_SWORD_RAZOR,
+    /* 0x39 */ GI_SWORD_GILDED,
+    /* 0x3A */ GI_BOMBCHUS_5,
+    /* 0x3B */ GI_SWORD_GREAT_FAIRY,
+    /* 0x3C */ GI_KEY_SMALL,
     /* 0x3D */ GI_KEY_BOSS,
     /* 0x3E */ GI_MAP,
     /* 0x3F */ GI_COMPASS,
-    /* 0x50 */ GI_50 = 0x50,
-    /* 0x52 */ GI_SCALE_GOLD = 0x52, // Assumed, used in En_Fishing
-    /* 0x59 */ GI_BOTTLE_POTION_RED = 0x59,
+    /* 0x41 */ GI_HOOKSHOT = 0x41,
+    /* 0x42 */ GI_LENS,
+    /* 0x43 */ GI_PICTO_BOX,
+    /* 0x4C */ GI_OCARINA = 0x4C,
+    /* 0x50 */ GI_BOMBERS_NOTEBOOK = 0x50,
+    /* 0x52 */ GI_SKULL_TOKEN = 0x52,
+    /* 0x55 */ GI_REMAINS_ODOLWA = 0x55,
+    /* 0x56 */ GI_REMAINS_GOHT,
+    /* 0x57 */ GI_REMAINS_GYORG,
+    /* 0x58 */ GI_REMAINS_TWINMOLD,
+    /* 0x59 */ GI_BOTTLE_POTION_RED, // ITEM_LONGSHOT
     /* 0x5A */ GI_BOTTLE,
     /* 0x5B */ GI_POTION_RED,
     /* 0x5C */ GI_POTION_GREEN,
     /* 0x5D */ GI_POTION_BLUE,
     /* 0x5E */ GI_FAIRY,
-    /* 0x60 */ GI_MILK_BOTTLE = 0x60,
+    /* 0x5F */ GI_DEKU_PRINCESS,
+    /* 0x60 */ GI_MILK_BOTTLE,
+    /* 0x61 */ GI_MILK_HALF,
+    /* 0x62 */ GI_FISH,
+    /* 0x63 */ GI_BUG,
+    /* 0x64 */ GI_BLUE_FIRE,
+    /* 0x65 */ GI_POE,
+    /* 0x66 */ GI_BIG_POE,
+    /* 0x67 */ GI_SPRING_WATER,
+    /* 0x68 */ GI_HOT_SPRING_WATER,
+    /* 0x69 */ GI_ZORA_EGG,
+    /* 0x6A */ GI_GOLD_DUST,
+    /* 0x6B */ GI_MUSHROOM,
+    /* 0x6E */ GI_SEA_HORSE = 0x6E,
+    /* 0x6F */ GI_BOTTLE_CHATEAU,
```

Comment:
Just want to make sure that these distinguish between "item and a bottle" and "item by itself".

Reply:
Hmm, above we have `GI_MILK_BOTTLE`. I feel like either this should match that as `GI_CHATEAU_BOTTLE`, or the milks match this. Same with the red potion bottle.